deployer/src/lib.rs

```diff
@@ -351,6 +351,9 @@ pub struct StartOptions {
     /// Add custom RUST_LOG_SILENCE env to all containers.
     #[clap(long)]
     rust_log_silence: Option<String>,
+    /// Mount host dev and udev.
+    #[clap(long, default_value = "false")]
```

Comment:
default should be false already

```suggestion
    #[clap(long)]
```